core/pom.xml

```diff
@@ -147,6 +147,51 @@
           </archive>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>3.0.1</version>
+        <executions>
+          <execution>
+            <id>copy-resources</id>
+            <phase>package</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>../eclipse_plugin/lib</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>target</directory>
+                  <include>${project.artifactId}-${project.version}.jar</include>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>2.10</version>
+        <executions>
+          <execution>
+            <id>copy-dependencies</id>
+            <phase>package</phase>
+            <goals>
+              <goal>copy-dependencies</goal>
```

Comment:
I saw that behaviour before. A full `maven clean install` should work fine, as can be seen at the Travis and AppVeyor CI runs.
I thought about using a script as well, but a) Maven already knows about the dependencies (and where to copy them from) and b) Maven should do w/o the manual copy step. Somehow.

Any Maven expert here to rescue?